quickstart.sh

```diff
@@ -353,6 +349,69 @@ The Parity client has started and is running in background!
 EOF
 }
 
+function checkCredentials() {
+  local url
+  local credentials
+  local http_code
+  url=$1
+  credentials=$2
+  http_code=$(curl --silent -o /dev/null -u "${credentials}" "${url}" -w '%{http_code}')
+  if [[ ${http_code} = "200" ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+function setupNetstatsInteractive() {
+  printmsg <<EOF
+We can setup a netstats client that reports to the netstats server
+running at
+
+  https://laikanetstats.trustlines.foundation/
+
+You will need credentials to do that.
+
+EOF
+  if [[ "$(askYesOrNo 'Did you already receive credentials and want to setup the netstats client?')" = "no" ]]; then
+    return 0
+  fi
+  while true; do
+    local username
+    local password
+    read -r -p "Username: " username
+    read -s -r -p "Password: " password
+    if checkCredentials https://laikanetstats.trustlines.foundation/check/ "${username}:${password}" ; then
+      printmsg <<EOF
+The provided credentials do work. Please enter an instance name
+now. You are free to choose any name you like.
+EOF
+      break
+    else
+      printmsg <<EOF
+The provided credentials do not work. Please try to enter them again.
+EOF
+    fi
+  done
+
+  read -r -p "Instance name: " instance_name
+  cat >"${NETSTATS_ENV_FILE}" <<EOF
+WS_USER=${username}
+WS_PASSWORD=${password}
+INSTANCE_NAME=${instance_name}
+# HIDE_VALIDATOR_STATUS=false
+EOF
+}
+
+function startNetstats() {
+  stopAndRemoveContainer ${DOCKER_CONTAINER_NETSTATS}
+  printmsg <<EOF
+Starting netstats container...
+EOF
+  ${PERMISSION_PREFIX} docker run --network ${DOCKER_NETWORK} --name ${DOCKER_CONTAINER_NETSTATS} -d --restart=always --env-file "${NETSTATS_ENV_FILE}" -e RPC_HOST=${DOCKER_CONTAINER_PARITY} ${DOCKER_IMAGE_NETSTATS}
```

Comment:
```suggestion
  ${PERMISSION_PREFIX} docker run \
    --detach \
    --restart=always \
    --network ${DOCKER_NETWORK} \
    --name ${DOCKER_CONTAINER_NETSTATS} \
    --env-file "${NETSTATS_ENV_FILE}" \
    --env RPC_HOST=${DOCKER_CONTAINER_PARITY} \
    ${DOCKER_IMAGE_NETSTATS}
```

Reply:
Writing out parameter names and split way too long command to multiple lines for readability.